_posts/2016-04-07-extension-approach-2.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+cover: 'assets/images/general-cover-3.jpg'
+title: Overview of Extension Approach 2 
+tags: docs
+subclass: 'post tag-docs'
+categories: 'elixir'
+author: 'Nimish'
+navigation: true
+logo: 'assets/images/nectar-cart.png'
+---
+>
+The post belongs to _NectarCommerce and Extension Framework Awareness_ Series
+>
+1. _[NectarCommerce Vision](http://vinsol.com/blog/2016/04/08/nectarcommerce-vision/)_
+1. _[Extension Framework Game Plan](http://vinsol.com/blog/2016/04/12/extension-framework-game-plan/)_
+1. _[Introduction to Metaprogramming](http://vinsol.com/blog/2016/04/14/introduction-to-metaprogramming/)_
+1. _[Ecto Model Schema Extension](http://vinsol.com/blog/2016/04/15/ecto-model-schema-extension/)_
+1. Ecto Model Support Functions Extension
+1. Phoenix Router Extension
+1. Phoenix View Extension
+1. Running Multiple Elixir Apps Together
+1. **Extension Approach Explained**
+1. Developer Experience and Workflow developing Favorite Product Extension
+1. Developer Experience and Workflow testing Favorite Product Extension
+
+## What will be NectarCommerce
+
+>
+Off-the-shelf Opensource E-commerce application for building an online store.
+>
+Provides an Extension Framework to support features not included in core as extensions.
+>
+Strives for unobtrusive parallel development of NectarCommerce and Extensions
+
+NectarCommerce is committed to providing a ready-to-use e-commerce solution but the definition of 100% is different under different business domains. It aims to solve common use-cases as part of the project and relying on extension framework to tap the rest.
+
+## NectarCommerce Extension Approach 2 ##
+
+This post aims at documenting how extensions approach 2 is structured, the order of dependencies amongst the various nuts and bolts of NectarCommerce and its extensions as implemented.
```

Comment:
Lets use one of these :  'Extensions approach'  or 'Extension approach'.  In the heading we say 'Extension approach' and in this line we are using 'Extensions approach'